source/install/desktop.rst

```diff
@@ -2,42 +2,42 @@
 Desktop Application Install Guides
 ===================================
 
-Mattermost desktop applications are available for Windows, Mac and Linux operating systems. They support all the features of the web experience, plus:
+Mattermost desktop applications are available for Windows, Mac, and Linux operating systems. They support all the features of the web experience, plus:
 
  - Connect to multiple Mattermost servers from a single interface, and switch with shortcut keys
  - Auto-start Mattermost when a user logs into their machine
- - (Windows) Add Mattermost to Start menu, taskbar and system tray
+ - (Windows) Add Mattermost to Start menu, Taskbar, and System Tray
  - (Windows/Mac) Deep link to the desktop app via mattermost:// protocol if app is already installed
- - (Mac) Add Mattermost to the applications Dock
+ - (Mac) Add Mattermost to the Dock
  - (Linux) ``Desktop Entry`` for the application to more easily `integrate into a desktop environment <https://wiki.archlinux.org/index.php/Desktop_entries>`__
 
 Below is a list of additional resources:
 
  - `Guide for configuring your desktop app experience <https://docs.mattermost.com/help/apps/desktop-guide.html>`__
  - `Changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`__
- - Contributor’s guide (coming soon)
  - `Source code <https://github.com/mattermost/desktop>`__
+ - Contributor’s guide (coming soon)
 
-You can `download the apps directly from our downloads page <https://about.mattermost.com/downloads/>`__. You may also use the following installation guides for Windows, Mac and Linux.
+You can `download the apps directly from our downloads page <https://about.mattermost.com/downloads/>`__. You may also use the following installation guides for Windows, Mac, and Linux.
 
 .. contents::
     :backlinks: top
 
 Windows 10+, Windows 8.1+, Windows 7+
---------------------------------------------------
+--------------------------------------
 
-1. Download latest version of the Mattermost desktop app:
+1. Download the latest version of the Mattermost desktop app:
 
    - `32/64-bit version of Windows <https://releases.mattermost.com/desktop/4.5.0/mattermost-desktop-setup-4.5.0-win.exe>`__
 
-2. From the ``\Downloads`` directory right-click on the file ``mattermost-desktop-setup-4.5.0-win.exe`` and select **Open**.
+2. From the **\Downloads** folder right-click on the file ``mattermost-desktop-setup-4.5.0-win.exe`` and select **Open**.
 
 This will start an installer for the app. Once finished, the Mattermost desktop app will open automatically.
 
 MSI Installer and Group Policies (Beta)
```

Comment:
Unrelated to this PR, but there is a broken link at ``See here for instructions on installing the Mattermost desktop app via an MSI installer and configuring supported Group Policies.``

The link should be: https://docs.mattermost.com/install/desktop-msi-gpo.html.

Reply:
Thanks! I've fixed that.